runtime/android/core/src/org/xwalk/core/XWalkUpdater.java

```diff
@@ -177,6 +158,13 @@
  * as the downloaded runtime Apk will be saved into application's private storage and then get
  * deleted after done extracting the libraries/resources from the runtime Apk.</p>
  *
+ * <p>But you might need to add following persmission to hide the notification icon when
+ * downloading.</p>
+ *
+ * <pre>
+ * &lt;uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" /&gt;
+ * </pre>
+ *
```

Comment:
hide notification -- WRITE_EXTERNAL_STORAGE? In download mode we don't show any download notification icon since we are using our own HTTP download instead of Android DownloadManager, so we may need to explain that the notification is only shown in shared mode runtime download.